test/addons/rook-cephfs/test

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import sys
+
+from drenv import kubectl
+
+PVC_NAME = "cephfs-pvc"
+SNAP_NAME = "cephfs-snap"
+
+
+def test_provisioning(cluster):
+    print(f"Deploying pvc {PVC_NAME} in cluster '{cluster}'")
+    kubectl.apply(
+        f"--filename={PVC_NAME}.yaml",
+        "--namespace=rook-ceph",
+        context=cluster,
+    )
+
+    print(f"Waiting until pvc {PVC_NAME} is bound in cluster '{cluster}'")
+    kubectl.wait(
+        f"pvc/{PVC_NAME}",
+        "--for=jsonpath={.status.phase}=Bound",
+        "--namespace=rook-ceph",
+        "--timeout=300s",
+        context=cluster,
+    )
+
+    print(f"Deploying snapshot {SNAP_NAME} in cluster '{cluster}'")
+    kubectl.apply(
+        f"--filename={SNAP_NAME}.yaml",
+        "--namespace=rook-ceph",
+        context=cluster,
+    )
+
+    print(f"Waiting until snapshot {SNAP_NAME} is bound in cluster '{cluster}'")
+    kubectl.wait(
+        f"volumesnapshot/{SNAP_NAME}",
+        "--for=jsonpath={.status.readyToUse}=true",
+        "--namespace=rook-ceph",
+        "--timeout=300s",
+        context=cluster,
+    )
+
+    print(f"Deleting snapshot {SNAP_NAME} in cluster '{cluster}'")
+    kubectl.delete(
+        f"volumesnapshot/{SNAP_NAME}",
+        "--namespace=rook-ceph",
+        context=cluster,
+    )
+
+    print(f"Deleting pvc {PVC_NAME} in cluster '{cluster}'")
+    kubectl.delete(
+        f"pvc/{PVC_NAME}",
+        "--namespace=rook-ceph",
+        context=cluster,
+    )
+
+    print(f"CephFS based PVC and Snapshot provisioning on cluster '{cluster}' succeeded")
```

Comment:
So there is an 88 char line length limitation in the python files? I see [this](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#labels-line-length) from black, which suggests so.

@nirs Considering we have a longer line length limitation overall in the project for the go files, I suggest we change this and add the `--line-lenght` parameter to the run of black? Having different line length requirement across does not make it intuitive when writing code.

Reply:
I don't think it is a problem, people using python expects the black default line length.

When using black you don't need to think about formatting or line length. You configure your editor to run Black on save (see .vimrc.example), and Black format the file for you. There is no need to discuss formatting issues.

Another solution is editorconfig (https://editorconfig.org/). We can add editor config file to the project specifying how various files and directories should be formatted, and the editors use the config to format.

If we want common line length for the entire project, we cannot use the current Go default (120) because it is bad for readability. I think Black default of 88 is good compromise so I'm happy to adopt it as the project line length.

Reply:
I believe we should have a line length for the project not parts of it.

We are on 120 and I think we continue that, as before I am not debating line length, it is 120 for this project!

For now I am backing off changing the black defaults, but if this becomes an issue repeatedly I will change the default for black.